qt/aqt/addons.py

```diff
@@ -132,6 +133,7 @@ def from_json_meta(dir_name: str, json_meta: Dict[str, Any]) -> AddonMeta:
             min_point_version=json_meta.get("min_point_version", 0) or 0,
             max_point_version=json_meta.get("max_point_version", 0) or 0,
             branch_index=json_meta.get("branch_index", 0) or 0,
+            human_version=json_meta.get("human_version", ""),
```

Comment:
Because write_addon_meta() checks for None, the second argument should be omitted here. I'll fix it after merging.